bower.json

```diff
@@ -25,7 +25,11 @@
     "paper-radio-button": "PolymerElements/paper-radio-button#^2.0.0",
     "paper-tabs": "PolymerElements/paper-tabs#^2.0.0",
     "paper-toggle-button": "PolymerElements/paper-toggle-button#^2.0.0",
-    "polymer": "Polymer/polymer#^2.0.0"
+    "polymer": "Polymer/polymer#^2.0.1"
```

Comment:
Was Polymer not resolved properly?
^ should have included everything greater than version 2.0.0 in the same major range, including patches.